cranetoolbox/importTools/transform.py

```diff
@@ -221,30 +234,54 @@ def parse_tweet(tweet: str) -> dict:
     return json.loads(tweet)
 
 
-def lighten_tweet(tweet: dict) -> (str, str, str):
+def lighten_tweet(tweet: dict, text_name, id_name, date_name) -> (
+str, str, str):
     """Lighten a tweet by returning only the fields required for analysis.
 
     :param tweet: A parsed JSON tweet
     :type tweet: dict
+    :param text_name : A string to different text field name
+    :type text_name: string
+    :param id_name : A string to different id field name
+    :type id_name: string
+    :param date_name : A string to different date field name
+    :type date_name: string
     :return: A tuple of the three values scraped from the passed tweet
     :rtype: tuple(str, str, str)
     """
 
-    created_at = tweet.get("created_at", None)
-    tweet_id = tweet.get("id", None)
+    if date_name is not None:
+        created_at = tweet.get(date_name, None)
+    else:
+        created_at = tweet.get("created_at", None)
+
+    if id_name is not None:
+        tweet_id = tweet.get(id_name, None)
+    else:
+        tweet_id = tweet.get("id", None)
+
     text = ""
+
     if tweet.get("truncated", False):
         ext_tweet = tweet.get("extended_tweet", None)
         if ext_tweet is not None:
-            text = ext_tweet.get("full_text", "")
```

Comment:
Misunderstanding there, for now we're not giving the user the possibility to specify _advanced_ fields so "full_text" will always be "full_text"

Reply:
in line 270, case the text field it not none, is joined the prefix "full_" with custom text field defined by user, then, if an user specific a text field like "comment" and the tweet is extended, then the field "ful_comment" will be search in tweet. Did you have think in other way?

Reply:
I understood that this is what your code does, but I'd prefer for the field to always to "full_text", even if the user renames the regular text field to "comment". Would that be okay?

Reply:
No problems, i will make the changes!